src/base/static/components/molecules/place-list-item.js

```diff
@@ -0,0 +1,269 @@
+import React from "react";
+import styled from "react-emotion";
+import PropTypes from "prop-types";
+import { Button, IconButton } from "../atoms/buttons";
+import { HeartIcon } from "../atoms/icons";
+import { SmallTitle } from "../atoms/typography";
+import { UserAvatar } from "../atoms/imagery";
+import { RegularText, SmallText, Link } from "../atoms/typography";
+import { placeConfigSelector } from "../../state/ducks/place-config";
+import { supportConfigSelector } from "../../state/ducks/support-config";
+import { appConfigSelector } from "../../state/ducks/app-config";
+import { connect } from "react-redux";
+import { translate } from "react-i18next";
+import { HorizontalRule } from "../atoms/layout";
+import sharePlace from "../../utils/share-place";
+
+const PlaceContainer = styled("div")({
+  display: "flex",
+  flexDirection: "column",
+  overflow: "hidden",
+  margin: "0px 16px 16px 16px",
+});
+
+const Header = styled("div")({
+  display: "flex",
+  flexDirection: "row",
+  alignContent: "space-between",
+});
+
+const Body = styled("div")({
+  display: "flex",
+});
+
+const PlaceInfo = styled("div")({
+  display: "flex",
+  flex: "0 30%",
+  marginRight: "16px",
+});
+const AvatarContainer = styled("div")({
+  flex: "0 30%",
+  minWidth: "48px",
+  marginRight: "8px",
+});
+
+const PlaceInfoContainer = styled("div")({
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "space-between",
+});
+const PlaceInfoTop = styled("div")({
+  alignItems: "start",
+  display: "flex",
+  flexDirection: "column",
+});
+const PlaceInfoButton = styled(Link)({
+  alignItems: "end",
+  marginTop: "16px",
+});
+
+const PlaceContent = styled("div")({
+  flex: "1 70%",
+  display: "flex",
+});
+
+const PlaceTitle = styled(SmallTitle)({
+  display: "flex",
+  flex: "1 60%",
+});
+
+const PlaceSocialContainer = styled("div")({
+  display: "flex",
+  flex: "0 40%",
+  maxWidth: "160px",
+  justifyContent: "flex-end",
+  alignItems: "center",
+});
+
+const SocialMediaButton = styled(IconButton)({
+  flex: "0 1",
+  marginLeft: "16px",
+});
+
+// Place Content components:
+const PlaceImage = styled("div")({
+  flex: "0 1 25%",
+  marginRight: "16px",
+});
+const PlaceFieldsContainer = styled("div")({
+  flex: "1 75%",
+  display: "flex",
+  flexDirection: "column",
+});
+const PlaceFieldTitle = styled(SmallTitle)({
+  width: "100%",
+});
+const PlaceFieldText = styled(SmallText)({
```

Comment:
It turns out that `SmallText` is pretty small. I think this would be more readable as `RegularText`.